changes/3444.bugfix.rst

```diff
@@ -0,0 +1 @@
+Ensure the startup method returns valid content by raising a ValueError when None is returned
```

Comment:
The changenote isn't a commit message - it's for the benefit of users downloading a new release so they know what bugs have been fixed in the new release.
```suggestion
Apps that use a function-based app startup method now validate that the startup method returns content that can be added to the main window.
```